runtimes/nodejs/object-delete.js

```diff
@@ -11,9 +11,25 @@
  *   }
  */
 var CloudObjectStorage = require('ibm-cos-sdk');
-function main(args) {
-  let { cos, params } = getParamsCOS(args, CloudObjectStorage);
-  return cos.deleteObject(params).promise();
+
+async function main(args) {
+  const { cos, params } = getParamsCOS(args, CloudObjectStorage);
+
+  let response;
+  try {
+    response = await cos.deleteObject(params).promise();
+  } catch (err) {
+    return Promise.reject({
+      Bucket: params.Bucket,
+      Key: params.Key,
+      Error: err,
```

Comment:
Field names should be lowercase.

Reply:
lower cased the returned field values - left params.Bucket & params.Key as capitalized as we pass those to COS, and COS needs capitalized for these keys names.